test/unit_tests/organisms/Footer-spec.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+var BASE_JS_PATH = '../../../cfgov/unprocessed/js/';
+
+var chai = require( 'chai' );
+var expect = chai.expect;
+//var Footer = require( BASE_JS_PATH + 'organisms/Footer' );
```

Comment:
Whoops, I forgot to go back to that one, it was causing errors due to the footer to top button. I'll get a fix pushed shortly.